src/main/java/com/rultor/agents/daemons/EndsDaemon.java

```diff
@@ -155,6 +168,30 @@ public String apply(final String str) {
                     )
                 )
             );
+        EndsDaemon.deprecate(directives, deprecate);
+        return directives;
+    }
+
+    /**
+     * Adds the deprecated message to the directive.
+     * @param directives the directives which being deprecated
+     * @param deprecate true if it is deprecated.
+     */
+    private static void deprecate(final Directives directives,
+        final boolean deprecate) {
+        if (deprecate) {
```

Comment:
@mkordas I will add a puzzle for the test. Actually, I just had added the test with a puzzle and closed the issue :) Next time i will work that way.